### PR TITLE
Support white fade

### PR DESCRIPTION
**Pull request by therealolivierl**
There seems to be a ton of indent changes: the real changes are here:

https://github.com/dakis/image/compare/master...dakis:supportWhiteFade?expand=1#diff-9452939f2ca4b63fbe7d0e3408c6f95bR228

### CODE REVIEW

**Comment by phacops** [comment]
Use `go fmt` on your project.


On Fri, Aug 19, 2016, at 09:14, therealolivierl wrote:
> There seems to be a ton of indent changes: the real changes are here:
> https://github.com/dakis/image/compare/master...dakis:supportWhiteFade?expand=1#diff-9452939f2ca4b63fbe7d0e3408c6f95bR228
>
> You can view, comment on, or merge this pull request online at:
>   https://github.com/dakis/image/pull/1
> Commit Summary


>  * test for white fade
>  * apart from identation change, SetWhiteFade now uses its parameter
>    in order to fade image
> File Changes


>  * *M* image.go[1] (385)
> Patch Links:


>  * https://github.com/dakis/image/pull/1.patch
>  * https://github.com/dakis/image/pull/1.diff
> — You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub[2], or mute the
> thread[3].
>

Links:

  1. https://github.com/dakis/image/pull/1/files#diff-0
  2. https://github.com/dakis/image/pull/1

File: image.go [hunk 1]
```diff
@@ -225,6 +226,21 @@ func (i *Image) SetGrayscale() (err error) {
 	return i.wand.SetImageType(imagick.IMAGE_TYPE_GRAYSCALE)
 }
 
+func (i *Image) SetWhiteFade(targetWhiteFade string) (err error) {
+	colorize := imagick.NewPixelWand()
+	opacity := imagick.NewPixelWand()
+	var color string = fmt.Sprintf("hsl(0, 0%%, %s%%)", targetWhiteFade)
+
+	log.Println(color)
```
Comment (phacops) [inline image.go]: You should remove the log. A librairie shouldn't log anything.

**Comment by phacops** [comment]
One comment to fix otherwise :+1:

**Comment by louiscloutierdakis** [comment]
+1

**Pull request merged**